applications/machine_learning/src/modules/ei_data_forwarder_uart.c

```diff
@@ -122,7 +122,7 @@ static bool handle_sensor_event(const struct sensor_event *event)
 	int err = uart_tx(dev, buf, pos, SYS_FOREVER_MS);
 
 	if (err) {
-		atomic_cas(&uart_busy, true, false);
+		(void)atomic_cas(&uart_busy, true, false);
```

Comment:
I am wondering why atomic_cas if we don't care about return value. In bool case it should make no difference then performing simple set.